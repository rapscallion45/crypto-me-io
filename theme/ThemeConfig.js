import PropTypes from 'prop-types';
import { useMemo } from 'react';
import { CacheProvider } from '@emotion/react';
import { CssBaseline } from '@mui/material';
import { ThemeProvider, createTheme } from '@mui/material/styles';
import shape from './shape';
import palette from './palette';
import typography from './typography';
import componentsOverride from './overrides';
import shadows, { customShadows } from './shadows';

const ThemeConfig = function Themeconfig({ emotionCache, children }) {
  const themeOptions = useMemo(
    () => ({
      palette,
      shape,
      typography,
      shadows,
      customShadows,
    }),
    []
  );

  const theme = createTheme(themeOptions);
  theme.components = componentsOverride(theme);

  return (
    <CacheProvider value={emotionCache}>
      <ThemeProvider theme={theme}>
        <CssBaseline />
        {children}
      </ThemeProvider>
    </CacheProvider>
  );
};

ThemeConfig.propTypes = {
  children: PropTypes.node,
};

export default ThemeConfig;
