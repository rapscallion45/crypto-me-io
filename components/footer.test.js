import React from 'react';
import { shallow } from 'enzyme';
import { render } from '@testing-library/react';
import toJson from 'enzyme-to-json';
import createEmotionCache from '../utils/createEmotionCache';
import ThemeConfig from '../theme/ThemeConfig';
import Footer from './footer';

describe('Footer', () => {
  it('Renders correctly enzyme', () => {
    /* Arrange */
    /* Act */
    const wrapper = shallow(
      <ThemeConfig emotionCache={createEmotionCache()}>
        <Footer />
      </ThemeConfig>
    );

    /* Assert */
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  describe('Copyright date', () => {
    it('Should render this year', () => {
      /* Arrange */
      const today = new Date();
      const date = today.getFullYear();

      /* Act */
      const wrapper = render(
        <ThemeConfig emotionCache={createEmotionCache()}>
          <Footer />
        </ThemeConfig>
      );

      /* Assert */
      expect(wrapper.getByText(date)).toBeInTheDocument();
    });
  });

  describe('Home link', () => {
    it('Should render link to homepage', () => {
      /* Arrange */
      const linkText = 'CryptoMe.io';
      const linkUrl = '/';

      /* Act */
      const wrapper = render(
        <ThemeConfig emotionCache={createEmotionCache()}>
          <Footer />
        </ThemeConfig>
      );

      /* Assert */
      expect(wrapper.getByText(linkText).closest('a')).toHaveAttribute('href', linkUrl);
    });
  });
});
