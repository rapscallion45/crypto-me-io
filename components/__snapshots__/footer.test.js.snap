// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders correctly enzyme 1`] = `
<ContextProvider
  value={
    Object {
      "insert": [Function],
      "inserted": Object {},
      "key": "css",
      "nonce": undefined,
      "registered": Object {},
      "sheet": StyleSheet {
        "_insertTag": [Function],
        "before": null,
        "container": <head />,
        "ctr": 0,
        "insertionPoint": undefined,
        "isSpeedy": false,
        "key": "css",
        "nonce": undefined,
        "prepend": undefined,
        "tags": Array [],
      },
    }
  }
>
  <ThemeProvider
    theme={
      Object {
        "breakpoints": Object {
          "between": [Function],
          "down": [Function],
          "keys": Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ],
          "only": [Function],
          "unit": "px",
          "up": [Function],
          "values": Object {
            "lg": 1200,
            "md": 900,
            "sm": 600,
            "xl": 1536,
            "xs": 0,
          },
        },
        "components": Object {
          "MuiAutocomplete": Object {
            "styleOverrides": Object {
              "paper": Object {
                "boxShadow": "0 0 2px 0 rgba(145, 158, 171, 0.24), 0 20px 40px -4px rgba(145, 158, 171, 0.24)",
              },
            },
          },
          "MuiBackdrop": Object {
            "styleOverrides": Object {
              "root": Object {
                "&.MuiBackdrop-invisible": Object {
                  "background": "transparent",
                },
                "background": Array [
                  "rgb(22,28,36)",
                  "-moz-linear-gradient(75deg, rgba(22, 28, 36, 0.48) 0%, rgba(22, 28, 36, 1) 100%)",
                  "-webkit-linear-gradient(75deg, rgba(22, 28, 36, 0.48) 0%, rgba(22, 28, 36, 1) 100%)",
                  "linear-gradient(75deg, rgba(22, 28, 36, 0.48) 0%, rgba(22, 28, 36, 1) 100%)",
                ],
              },
            },
          },
          "MuiBottomNavigation": Object {
            "styleOverrides": Object {
              "root": Object {
                "borderTopColor": "#ffffff",
                "borderTopStyle": "solid",
                "borderTopWidth": "1px",
                "bottom": 0,
                "boxShadow": "rgb(0 0 0 / 20%) 0px 2px 4px -1px, rgb(0 0 0 / 14%) 0px 4px 5px 0px, rgb(0 0 0 / 12%) 0px 1px 10px 0px",
                "height": "75px",
                "position": "fixed",
                "width": "100%",
              },
            },
          },
          "MuiBottomNavigationAction": Object {
            "styleOverrides": Object {
              "label": Object {
                "&.Mui-selected": Object {
                  "fontSize": "0.75rem",
                },
              },
              "root": Object {
                "&.Mui-selected": Object {
                  "paddingTop": "12px",
                },
                "color": "#c0c0c0",
                "minWidth": "40px",
                "paddingTop": "12px",
              },
            },
          },
          "MuiButton": Object {
            "styleOverrides": Object {
              "containedInherit": Object {
                "&:hover": Object {
                  "backgroundColor": "#C4CDD5",
                },
                "boxShadow": "0 8px 16px 0 rgba(145, 158, 171, 0.24)",
                "color": "#212B36",
              },
              "containedPrimary": Object {
                "&:hover": Object {
                  "backgroundColor": "#0033e7",
                  "color": "#00FF87",
                },
              },
              "containedSecondary": Object {
                "&:hover": Object {
                  "backgroundColor": "white",
                  "color": "#919EAB",
                },
                "backgroundColor": "white",
                "color": "#919EAB",
              },
              "outlinedInherit": Object {
                "&:hover": Object {
                  "backgroundColor": "rgba(145, 158, 171, 0.08)",
                },
                "border": "1px solid rgba(145, 158, 171, 0.32)",
              },
              "root": Object {
                "&:hover": Object {
                  "boxShadow": "none",
                },
                "border": "2px solid",
              },
              "sizeLarge": Object {
                "height": 48,
              },
              "textInherit": Object {
                "&:hover": Object {
                  "backgroundColor": "rgba(145, 158, 171, 0.08)",
                },
              },
            },
          },
          "MuiCard": Object {
            "styleOverrides": Object {
              "root": Object {
                "borderRadius": 16,
                "boxShadow": "0 0 2px 0 rgba(145, 158, 171, 0.24), 0 16px 32px -4px rgba(145, 158, 171, 0.24)",
                "position": "relative",
                "zIndex": 0,
              },
            },
          },
          "MuiCardContent": Object {
            "styleOverrides": Object {
              "root": Object {
                "padding": "24px",
              },
            },
          },
          "MuiCardHeader": Object {
            "defaultProps": Object {
              "subheaderTypographyProps": Object {
                "variant": "body2",
              },
              "titleTypographyProps": Object {
                "variant": "h6",
              },
            },
            "styleOverrides": Object {
              "root": Object {
                "padding": "24px 24px 0px",
              },
            },
          },
          "MuiChip": Object {
            "styleOverrides": Object {
              "colorSecondary": Object {
                "color": "inherit",
              },
              "label": Object {
                "paddingTop": "1px",
              },
              "root": Object {
                "borderRadius": "6px",
              },
            },
          },
          "MuiDialog": Object {
            "styleOverrides": Object {
              "paper": Object {
                "margin": "32px 10px",
                "width": "calc(100% - 20px)",
              },
            },
          },
          "MuiDialogContent": Object {
            "styleOverrides": Object {
              "root": Object {
                "padding": "8px",
              },
            },
          },
          "MuiFilledInput": Object {
            "styleOverrides": Object {
              "root": Object {
                "&.Mui-disabled": Object {
                  "backgroundColor": "rgba(145, 158, 171, 0.24)",
                },
                "&.Mui-focused": Object {
                  "backgroundColor": "rgba(145, 158, 171, 0.24)",
                },
                "&:hover": Object {
                  "backgroundColor": "rgba(145, 158, 171, 0.16)",
                },
                "backgroundColor": "rgba(145, 158, 171, 0.12)",
              },
              "underline": Object {
                "&:before": Object {
                  "borderBottomColor": "rgba(145, 158, 171, 0.56)",
                },
              },
            },
          },
          "MuiIconButton": Object {
            "styleOverrides": Object {
              "root": Object {},
            },
            "variants": Array [
              Object {
                "props": Object {
                  "color": "default",
                },
                "style": Object {
                  "&:hover": Object {
                    "backgroundColor": "rgba(145, 158, 171, 0.08)",
                  },
                },
              },
              Object {
                "props": Object {
                  "color": "inherit",
                },
                "style": Object {
                  "&:hover": Object {
                    "backgroundColor": "rgba(145, 158, 171, 0.08)",
                  },
                },
              },
            ],
          },
          "MuiInput": Object {
            "styleOverrides": Object {
              "underline": Object {
                "&:before": Object {
                  "borderBottomColor": "rgba(145, 158, 171, 0.56)",
                },
              },
            },
          },
          "MuiInputBase": Object {
            "styleOverrides": Object {
              "input": Object {
                "&::placeholder": Object {
                  "color": "#919EAB",
                  "opacity": 1,
                },
              },
              "root": Object {
                "&.Mui-disabled": Object {
                  "& svg": Object {
                    "color": "#919EAB",
                  },
                },
              },
            },
          },
          "MuiListItemAvatar": Object {
            "styleOverrides": Object {
              "root": Object {
                "marginRight": "16px",
                "minWidth": "auto",
              },
            },
          },
          "MuiListItemIcon": Object {
            "styleOverrides": Object {
              "root": Object {
                "color": "inherit",
                "marginRight": "16px",
                "minWidth": "auto",
              },
            },
          },
          "MuiListItemText": Object {
            "styleOverrides": Object {
              "multiline": Object {
                "marginBottom": 0,
                "marginTop": 0,
              },
              "root": Object {
                "marginBottom": 0,
                "marginTop": 0,
              },
            },
          },
          "MuiOutlinedInput": Object {
            "styleOverrides": Object {
              "root": Object {
                "& .MuiOutlinedInput-notchedOutline": Object {
                  "borderColor": "rgba(145, 158, 171, 0.32)",
                },
                "&.Mui-disabled": Object {
                  "& .MuiOutlinedInput-notchedOutline": Object {
                    "borderColor": "rgba(145, 158, 171, 0.24)",
                  },
                },
              },
            },
          },
          "MuiPaper": Object {
            "defaultProps": Object {
              "elevation": 0,
            },
            "styleOverrides": Object {
              "root": Object {
                "backgroundImage": "none",
              },
            },
          },
          "MuiSlider": Object {
            "styleOverrides": Object {
              "markLabel": Object {
                "top": "25px !important",
              },
              "root": Object {
                "padding": "13px 0 !important",
              },
            },
          },
          "MuiTable": Object {
            "styleOverrides": Object {
              "root": Object {
                "minWidth": "10px !important",
              },
            },
          },
          "MuiTableCell": Object {
            "styleOverrides": Object {
              "root": Object {
                "padding": "6px 6px",
                "verticalAlign": "middle",
                "width": "100px",
              },
            },
          },
          "MuiTableRow": Object {
            "styleOverrides": Object {
              "root": Object {
                "verticalAlign": "bottom",
              },
            },
          },
          "MuiTooltip": Object {
            "styleOverrides": Object {
              "arrow": Object {
                "color": "#212B36",
              },
              "tooltip": Object {
                "backgroundColor": "#212B36",
              },
            },
          },
          "MuiTypography": Object {
            "styleOverrides": Object {
              "gutterBottom": Object {
                "marginBottom": "8px",
              },
              "paragraph": Object {
                "marginBottom": "16px",
              },
            },
          },
        },
        "customShadows": Object {
          "error": "0 8px 16px 0 rgba(255, 72, 66, 0.24)",
          "info": "0 8px 16px 0 rgba(24, 144, 255, 0.24)",
          "primary": "0 8px 16px 0 rgba(0, 51, 231, 0.24)",
          "secondary": "0 8px 16px 0 rgba(255, 255, 255, 0.24)",
          "success": "0 8px 16px 0 rgba(0, 255, 135, 0.24)",
          "warning": "0 8px 16px 0 rgba(255, 193, 7, 0.24)",
          "z1": "0 1px 2px 0 rgba(145, 158, 171, 0.24)",
          "z12": "0 0 2px 0 rgba(145, 158, 171, 0.24), 0 12px 24px 0 rgba(145, 158, 171, 0.24)",
          "z16": "0 0 2px 0 rgba(145, 158, 171, 0.24), 0 16px 32px -4px rgba(145, 158, 171, 0.24)",
          "z20": "0 0 2px 0 rgba(145, 158, 171, 0.24), 0 20px 40px -4px rgba(145, 158, 171, 0.24)",
          "z24": "0 0 4px 0 rgba(145, 158, 171, 0.24), 0 24px 48px 0 rgba(145, 158, 171, 0.24)",
          "z8": "0 8px 16px 0 rgba(145, 158, 171, 0.24)",
        },
        "direction": "ltr",
        "mixins": Object {
          "toolbar": Object {
            "@media (min-width:0px) and (orientation: landscape)": Object {
              "minHeight": 48,
            },
            "@media (min-width:600px)": Object {
              "minHeight": 64,
            },
            "minHeight": 56,
          },
        },
        "palette": Object {
          "action": Object {
            "activatedOpacity": 0.12,
            "active": "#637381",
            "disabled": "rgba(145, 158, 171, 0.8)",
            "disabledBackground": "rgba(145, 158, 171, 0.24)",
            "disabledOpacity": 0.48,
            "focus": "rgba(145, 158, 171, 0.24)",
            "focusOpacity": 0.12,
            "hover": "rgba(145, 158, 171, 0.08)",
            "hoverOpacity": 0.08,
            "selected": "rgba(145, 158, 171, 0.16)",
            "selectedOpacity": 0.08,
          },
          "augmentColor": [Function],
          "background": Object {
            "default": "#fff",
            "neutral": "#F4F6F8",
            "paper": "#fff",
          },
          "chart": Object {
            "blue": Array [
              "#2D99FF",
              "#83CFFF",
              "#A5F3FF",
              "#CCFAFF",
            ],
            "green": Array [
              "#2CD9C5",
              "#60F1C8",
              "#A4F7CC",
              "#C0F2DC",
            ],
            "red": Array [
              "#FF6C40",
              "#FF8F6D",
              "#FFBD98",
              "#FFF2D4",
            ],
            "violet": Array [
              "#826AF9",
              "#9E86FF",
              "#D0AEFF",
              "#F7D2FF",
            ],
            "yellow": Array [
              "#FFE700",
              "#FFEF5A",
              "#FFF7AE",
              "#FFF3D6",
            ],
          },
          "common": Object {
            "black": "#000",
            "white": "#fff",
          },
          "contrastThreshold": 3,
          "divider": "rgba(145, 158, 171, 0.24)",
          "error": Object {
            "contrastText": "#fff",
            "dark": "#B72136",
            "darker": "#7A0C2E",
            "light": "#FF7672",
            "lighter": "#FFBDBB",
            "lightest": "#FFF2F2",
            "main": "#FF4842",
          },
          "getContrastText": [Function],
          "gradients": Object {
            "error": "linear-gradient(to bottom, #FF7672, #FF4842)",
            "info": "linear-gradient(to bottom, #74CAFF, #1890FF)",
            "primary": "linear-gradient(to bottom, #89568f, #0033e7)",
            "success": "linear-gradient(to bottom, #68FFB8, #00FF87)",
            "warning": "linear-gradient(to bottom, #FFE16A, #FFC107)",
          },
          "grey": Object {
            "0": "#FFFFFF",
            "100": "#F9FAFB",
            "200": "#F4F6F8",
            "300": "#DFE3E8",
            "400": "#C4CDD5",
            "50": "#fafafa",
            "500": "#919EAB",
            "50012": "rgba(145, 158, 171, 0.12)",
            "50016": "rgba(145, 158, 171, 0.16)",
            "50024": "rgba(145, 158, 171, 0.24)",
            "50032": "rgba(145, 158, 171, 0.32)",
            "50048": "rgba(145, 158, 171, 0.48)",
            "50056": "rgba(145, 158, 171, 0.56)",
            "5008": "rgba(145, 158, 171, 0.08)",
            "50080": "rgba(145, 158, 171, 0.8)",
            "600": "#637381",
            "700": "#454F5B",
            "800": "#212B36",
            "900": "#161C24",
            "A100": "#f5f5f5",
            "A200": "#eeeeee",
            "A400": "#bdbdbd",
            "A700": "#616161",
          },
          "info": Object {
            "contrastText": "#fff",
            "dark": "#0C53B7",
            "darker": "#04297A",
            "light": "#74CAFF",
            "lighter": "#D0F2FF",
            "main": "#1890FF",
          },
          "mode": "light",
          "primary": Object {
            "contrastText": "#fff",
            "dark": "#007B55",
            "darker": "#18001a",
            "light": "#89568f",
            "lighter": "#b084b5",
            "main": "#0033e7",
          },
          "secondary": Object {
            "contrastText": "#fff",
            "dark": "#1939B7",
            "darker": "#091A7A",
            "light": "#84A9FF",
            "lighter": "#D6E4FF",
            "main": "#ffffff",
          },
          "success": Object {
            "contrastText": "#212B36",
            "dark": "#00C569",
            "darker": "#008C4A",
            "light": "#68FFB8",
            "lighter": "#C3FFE3",
            "lightest": "#E4FFF2",
            "main": "#00FF87",
          },
          "text": Object {
            "disabled": "#919EAB",
            "primary": "#212B36",
            "secondary": "#637381",
          },
          "tonalOffset": 0.2,
          "warning": Object {
            "contrastText": "#212B36",
            "dark": "#B78103",
            "darker": "#7A4F01",
            "light": "#FFE16A",
            "lighter": "#FFF7CD",
            "main": "#FFC107",
          },
        },
        "shadows": Array [
          "none",
          "0px 2px 1px -1px rgba(145, 158, 171, 0.2),0px 1px 1px 0px rgba(145, 158, 171, 0.14),0px 1px 3px 0px rgba(145, 158, 171, 0.12)",
          "0px 3px 1px -2px rgba(145, 158, 171, 0.2),0px 2px 2px 0px rgba(145, 158, 171, 0.14),0px 1px 5px 0px rgba(145, 158, 171, 0.12)",
          "0px 3px 3px -2px rgba(145, 158, 171, 0.2),0px 3px 4px 0px rgba(145, 158, 171, 0.14),0px 1px 8px 0px rgba(145, 158, 171, 0.12)",
          "0px 2px 4px -1px rgba(145, 158, 171, 0.2),0px 4px 5px 0px rgba(145, 158, 171, 0.14),0px 1px 10px 0px rgba(145, 158, 171, 0.12)",
          "0px 3px 5px -1px rgba(145, 158, 171, 0.2),0px 5px 8px 0px rgba(145, 158, 171, 0.14),0px 1px 14px 0px rgba(145, 158, 171, 0.12)",
          "0px 3px 5px -1px rgba(145, 158, 171, 0.2),0px 6px 10px 0px rgba(145, 158, 171, 0.14),0px 1px 18px 0px rgba(145, 158, 171, 0.12)",
          "0px 4px 5px -2px rgba(145, 158, 171, 0.2),0px 7px 10px 1px rgba(145, 158, 171, 0.14),0px 2px 16px 1px rgba(145, 158, 171, 0.12)",
          "0px 5px 5px -3px rgba(145, 158, 171, 0.2),0px 8px 10px 1px rgba(145, 158, 171, 0.14),0px 3px 14px 2px rgba(145, 158, 171, 0.12)",
          "0px 5px 6px -3px rgba(145, 158, 171, 0.2),0px 9px 12px 1px rgba(145, 158, 171, 0.14),0px 3px 16px 2px rgba(145, 158, 171, 0.12)",
          "0px 6px 6px -3px rgba(145, 158, 171, 0.2),0px 10px 14px 1px rgba(145, 158, 171, 0.14),0px 4px 18px 3px rgba(145, 158, 171, 0.12)",
          "0px 6px 7px -4px rgba(145, 158, 171, 0.2),0px 11px 15px 1px rgba(145, 158, 171, 0.14),0px 4px 20px 3px rgba(145, 158, 171, 0.12)",
          "0px 7px 8px -4px rgba(145, 158, 171, 0.2),0px 12px 17px 2px rgba(145, 158, 171, 0.14),0px 5px 22px 4px rgba(145, 158, 171, 0.12)",
          "0px 7px 8px -4px rgba(145, 158, 171, 0.2),0px 13px 19px 2px rgba(145, 158, 171, 0.14),0px 5px 24px 4px rgba(145, 158, 171, 0.12)",
          "0px 7px 9px -4px rgba(145, 158, 171, 0.2),0px 14px 21px 2px rgba(145, 158, 171, 0.14),0px 5px 26px 4px rgba(145, 158, 171, 0.12)",
          "0px 8px 9px -5px rgba(145, 158, 171, 0.2),0px 15px 22px 2px rgba(145, 158, 171, 0.14),0px 6px 28px 5px rgba(145, 158, 171, 0.12)",
          "0px 8px 10px -5px rgba(145, 158, 171, 0.2),0px 16px 24px 2px rgba(145, 158, 171, 0.14),0px 6px 30px 5px rgba(145, 158, 171, 0.12)",
          "0px 8px 11px -5px rgba(145, 158, 171, 0.2),0px 17px 26px 2px rgba(145, 158, 171, 0.14),0px 6px 32px 5px rgba(145, 158, 171, 0.12)",
          "0px 9px 11px -5px rgba(145, 158, 171, 0.2),0px 18px 28px 2px rgba(145, 158, 171, 0.14),0px 7px 34px 6px rgba(145, 158, 171, 0.12)",
          "0px 9px 12px -6px rgba(145, 158, 171, 0.2),0px 19px 29px 2px rgba(145, 158, 171, 0.14),0px 7px 36px 6px rgba(145, 158, 171, 0.12)",
          "0px 10px 13px -6px rgba(145, 158, 171, 0.2),0px 20px 31px 3px rgba(145, 158, 171, 0.14),0px 8px 38px 7px rgba(145, 158, 171, 0.12)",
          "0px 10px 13px -6px rgba(145, 158, 171, 0.2),0px 21px 33px 3px rgba(145, 158, 171, 0.14),0px 8px 40px 7px rgba(145, 158, 171, 0.12)",
          "0px 10px 14px -6px rgba(145, 158, 171, 0.2),0px 22px 35px 3px rgba(145, 158, 171, 0.14),0px 8px 42px 7px rgba(145, 158, 171, 0.12)",
          "0px 11px 14px -7px rgba(145, 158, 171, 0.2),0px 23px 36px 3px rgba(145, 158, 171, 0.14),0px 9px 44px 8px rgba(145, 158, 171, 0.12)",
          "0px 11px 15px -7px rgba(145, 158, 171, 0.2),0px 24px 38px 3px rgba(145, 158, 171, 0.14),0px 9px 46px 8px rgba(145, 158, 171, 0.12)",
        ],
        "shape": Object {
          "borderRadius": 8,
          "borderRadiusMd": 16,
          "borderRadiusSm": 12,
        },
        "spacing": [Function],
        "transitions": Object {
          "create": [Function],
          "duration": Object {
            "complex": 375,
            "enteringScreen": 225,
            "leavingScreen": 195,
            "short": 250,
            "shorter": 200,
            "shortest": 150,
            "standard": 300,
          },
          "easing": Object {
            "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
            "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
            "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
            "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
          },
          "getAutoHeightDuration": [Function],
        },
        "typography": Object {
          "body1": Object {
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "body2": Object {
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "0.875rem",
            "fontWeight": 400,
            "lineHeight": 1.5714285714285714,
          },
          "body3": Object {
            "fontSize": "0.8em",
            "lineHeight": 1.5714285714285714,
          },
          "body4": Object {
            "fontSize": "0.7rem",
            "lineHeight": 1.5714285714285714,
          },
          "button": Object {
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "0.875rem",
            "fontWeight": 700,
            "lineHeight": 1.7142857142857142,
            "textTransform": "capitalize",
          },
          "caption": Object {
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "0.75rem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "fontFamily": "Work Sans, sans-serif",
          "fontSize": 14,
          "fontWeightBold": 700,
          "fontWeightLight": 300,
          "fontWeightMedium": 600,
          "fontWeightRegular": 400,
          "h1": Object {
            "@media (min-width:1200px)": Object {
              "fontSize": "4rem",
            },
            "@media (min-width:600px)": Object {
              "fontSize": "3.25rem",
            },
            "@media (min-width:900px)": Object {
              "fontSize": "3.625rem",
            },
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "2.5rem",
            "fontWeight": 700,
            "lineHeight": 1.25,
          },
          "h2": Object {
            "@media (min-width:1200px)": Object {
              "fontSize": "3rem",
            },
            "@media (min-width:600px)": Object {
              "fontSize": "2.5rem",
            },
            "@media (min-width:900px)": Object {
              "fontSize": "2.75rem",
            },
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "2rem",
            "fontWeight": 700,
            "lineHeight": 1.3333333333333333,
          },
          "h3": Object {
            "@media (min-width:1200px)": Object {
              "fontSize": "2rem",
            },
            "@media (min-width:600px)": Object {
              "fontSize": "1.625rem",
            },
            "@media (min-width:900px)": Object {
              "fontSize": "1.875rem",
            },
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "1.5rem",
            "fontWeight": 700,
            "lineHeight": 1.5,
          },
          "h4": Object {
            "@media (min-width:1200px)": Object {
              "fontSize": "1.5rem",
            },
            "@media (min-width:600px)": Object {
              "fontSize": "1.25rem",
            },
            "@media (min-width:900px)": Object {
              "fontSize": "1.5rem",
            },
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "1.25rem",
            "fontWeight": 700,
            "lineHeight": 1.5,
          },
          "h5": Object {
            "@media (min-width:1200px)": Object {
              "fontSize": "1.25rem",
            },
            "@media (min-width:600px)": Object {
              "fontSize": "1.1875rem",
            },
            "@media (min-width:900px)": Object {
              "fontSize": "1.25rem",
            },
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "1.125rem",
            "fontWeight": 700,
            "lineHeight": 1.5,
          },
          "h6": Object {
            "@media (min-width:1200px)": Object {
              "fontSize": "1.125rem",
            },
            "@media (min-width:600px)": Object {
              "fontSize": "1.125rem",
            },
            "@media (min-width:900px)": Object {
              "fontSize": "1.125rem",
            },
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "1.0625rem",
            "fontWeight": 700,
            "lineHeight": 1.5555555555555556,
          },
          "htmlFontSize": 16,
          "overline": Object {
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "0.75rem",
            "fontWeight": 700,
            "letterSpacing": 1.1,
            "lineHeight": 1.5,
            "textTransform": "uppercase",
          },
          "pxToRem": [Function],
          "subtitle1": Object {
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "1rem",
            "fontWeight": 600,
            "lineHeight": 1.5,
          },
          "subtitle2": Object {
            "fontFamily": "Work Sans, sans-serif",
            "fontSize": "0.875rem",
            "fontWeight": 600,
            "lineHeight": 1.5714285714285714,
          },
        },
        "zIndex": Object {
          "appBar": 1100,
          "drawer": 1200,
          "mobileStepper": 1000,
          "modal": 1300,
          "snackbar": 1400,
          "speedDial": 1050,
          "tooltip": 1500,
        },
      }
    }
  >
    <CssBaseline />
    <Footer />
  </ThemeProvider>
</ContextProvider>
`;
